lib/ansible/constants.py

```diff
@@ -99,6 +99,61 @@ def set_constant(name, value, export=vars()):
 VAULT_VERSION_MIN = 1.0
 VAULT_VERSION_MAX = 1.0
 
+# FIXME: remove once play_context mangling is removed
+MAGIC_VARIABLE_MAPPING = dict(
```

Comment:
why dict(k=v) instead of {k:v}? Is it faster? more readable? policy? better for python3? something else?
_ah. I see that this is just a move from play_context.py - I'm still curious about the difference in syntax. Do you know why?_

Reply:
i just copied, didn't reformat